services/ui-src/src/components/tables/Table.tsx

```diff
@@ -59,10 +59,10 @@ export const Table = ({
         {/* if content prop is passed, parse and render rows and cells */}
         {content.bodyRows &&
           content.bodyRows!.map((row: string[], index: number) => (
-            <Tr key={row[0] + index}>
+            <Tr key={`${row[0]}${index}-body-row`}>
```

Comment:
Fixes `Warning: Encountered two children with the same key, NaN. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.`.